ruby/ql/lib/codeql/ruby/ast/Call.qll

```diff
@@ -38,6 +41,7 @@ class Call extends Expr instanceof CallImpl {
    * foo :bar "baz", qux: 123
    * ```
    */
+  overlay[global]
```

Comment:
I think this predicate can reasonably be made local by replacing
```ql
p.getKey().getConstantValue().isSymbol(keyword)
```
below with
```ql
keyword = p.getKey().(SymbolLiteral).(StringlikeLiteralImpl).getStringValue()
```
(requires an `private import internal.Literal` import).